### DISCUSSION TITLE
Native events not firing inside DragSelect selectable area

### DISCUSSION THREAD

**Discussion by clementrcheng** [ThibaultJanBeyer/DragSelect#160] [Q&A]
When I define ‘area’ in the DragSelect settings, it seems that the non-selectable html elements that are inside the defined DragSelect area, which are listening to native events (click, mouse down, etc) are having those native events blocked or intercepted by DragSelect. They aren’t firing at all. This is only an issue on iPad Safari and Chrome, but not on Desktop. If I don’t define an area, all elements that aren’t selectable on page are having their events not fire. Anybody seen this before or have an idea what might be causing it?

Thanks in advance!

**Comment by ThibaultJanBeyer** [comment]
Hi @clementrcheng this sounds like a bug. Would you be so kind to [file an bug report](https://github.com/ThibaultJanBeyer/DragSelect/issues/new?assignees=&labels=&template=bug_report.md), fill in all the fields in the questionnaire and most importantly, add a code example so that we can try to reproduce it. Thank you!
